src/wizard-settings/header-bidding-gam/index.js

```diff
@@ -0,0 +1,284 @@
+										>
+											{ order.order_name }
+										</a>
+										{ ', ' }
+										{ order?.line_item_ids
+											? sprintf(
+													_n(
+														'containing %s line item',
+														'containing %s line items',
+														order.line_item_ids.length,
+														'newspack-ads'
+													),
+													order.line_item_ids.length
+											  )
+											: __( 'missing line items configuration' ) }
+										.
+									</span>
+								) : (
+									getMissingOrderMessage()
+								) }
+							</div>
+						) }
+					</Fragment>
+				) }
+				checkbox={ isValid() ? 'checked' : 'unchecked' }
+				actionText={ canConfigure() ? __( 'Configure', 'newspack-ads' ) : null }
+				onClick={ () => setIsCreating( true ) }
+			/>
+			{ isCreating && (
+				<Modal
+					title={ __( 'Create Order', 'newspack-ads' ) }
+					onRequestClose={ () => ! inFlight && setIsCreating( false ) }
+				>
+					<p>
+						{ __(
+							'Create the order and line items on your Google Ad Manager network according to the pre-defined price bucket settings.',
+							'newspack-ads'
+						) }
+					</p>
+					{ error && error.data?.status !== '404' && (
+						<Notice isError noticeText={ error.message } />
+					) }
+					<TextControl
+						label={ __( 'Order name', 'newspack-ads' ) }
+						disabled={ inFlight || order?.order_name }
+						value={ order?.order_name ? order.order_name : orderName }
+						onChange={ value => setOrderName( value ) }
+					/>
+					{ ! inFlight && order?.order_id && ! order?.line_item_ids?.length && (
+						<Notice
+							isWarning
+							noticeText={ __( "Order exists but it's missing its line items.", 'newspack-ads' ) }
+						/>
+					) }
+					{ ! inFlight &&
+						order?.order_id &&
+						order?.line_item_ids?.length &&
+						totalBatches > ( order?.lica_batch_count || 0 ) && (
+							<Notice
+								isWarning
+								noticeText={ __(
+									"Order and line items exists but it's missing the creatives associations.",
+									'newspack-ads'
+								) }
+							/>
+						) }
+					{ step && stepName ? (
+						<Fragment>
+							<Notice
+								isWarning
+								noticeText={ __(
+									'This may take up to 15 minutes, please do not close the window.',
+									'newspack-ads'
+								) }
+							/>
+							<ProgressBar completed={ step } total={ totalSteps } label={ stepName } />
+						</Fragment>
+					) : null }
+					<Card buttonsCard noBorder className="justify-end">
+						<Button
+							isSecondary
+							disabled={ inFlight }
+							onClick={ () => {
+								setIsCreating( false );
+							} }
+						>
+							{ __( 'Cancel', 'newspack-ads' ) }
+						</Button>
+						<Button isPrimary disabled={ ! orderName || inFlight } onClick={ create }>
+							{ order?.order_id
+								? __( 'Fix issues', 'newspack-ads' )
+								: __( 'Create Order', 'newspack-ads' ) }
+						</Button>
+					</Card>
+				</Modal>
+			) }
+		</Fragment>
+	);
+};
+
+wp.hooks.addFilter(
```

Comment:
We might consider importing from the `@wordpress/hooks` library instead, e.g. `import { applyFilters } from '@wordpress/hooks';`

Maybe this also warrants adding that library as a dependency to `newspack-scripts`, but that can be a separate thing.

Reply:
Agreed! Now that `newspack-scripts` is implemented, 7167e3a2edd260c8508627915750955d31c7bfc7 changes it to use `@wordpress/hooks`.